peps/pep-0736.rst

```diff
@@ -3,12 +3,13 @@ Title: Shorthand syntax for keyword arguments at invocation
 Author: Joshua Bambrick <jbambrick@google.com>,
         Chris Angelico <rosuav@gmail.com>
 Sponsor: Guido van Rossum <guido@python.org>
-Discussions-To: https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217
+Discussions-To: https://discuss.python.org/t/pep-736-shorthand-syntax-for-keyword-arguments-at-invocation/43432
 Status: Draft
 Type: Standards Track
 Created: 28-Nov-2023
 Python-Version: 3.13
-Post-History: 14-Oct-2023
+Post-History: `14-Oct-2023 <https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217>`__,
+              `18-Jan-2024 <https://discuss.python.org/t/pep-736-shorthand-syntax-for-keyword-arguments-at-invocation/43432>`__,
```

Comment:
```suggestion
              `17-Jan-2024 <https://discuss.python.org/t/pep-736-shorthand-syntax-for-keyword-arguments-at-invocation/43432>`__,
```

Reply:
Funny how timezones work :sweat_smile:
I guess 17th makes more sense given the other dates.